prov/terraform/openstack/osinstance.go

```diff
@@ -460,33 +460,14 @@ func computeNetworkAttributes(ctx context.Context, opts osInstanceOptions,
 	instance *ComputeInstance, outputs map[string]string) error {
 
 	log.Debugf("Looking for Network id for %q", networkNodeName)
-	var networkID string
-	resultChan := make(chan string, 1)
-	go func() {
-		for {
-			nID, err := deployments.GetInstanceAttributeValue(ctx, opts.deploymentID, networkNodeName, opts.instanceName, "network_id")
-			if err != nil {
-				log.Printf("[Warning] bypassing error while waiting for a network id: %v", err)
-			}
-			// As networkID is an optional property GetInstanceAttribute then GetProperty
-			// may return an empty networkID so keep checking as long as we have it
-			if nID != nil && nID.RawString() != "" {
-				resultChan <- nID.RawString()
-				return
-			}
-			select {
-			case <-time.After(1 * time.Second):
-			case <-ctx.Done():
-				// context cancelled, give up!
-				return
-			}
-		}
-	}()
-	select {
-	case networkID = <-resultChan:
-	case <-ctx.Done():
-		return ctx.Err()
+
+	// As Network instance is unique
+	defaultNetworkInstanceName := "0"
```

Comment:
I would implement this as a global variable (e.g. `DefaultInstanceName` or `FirstInstanceName`) in the deployments package instead, in case [this implementation](https://github.com/ystia/yorc/blob/develop/deployments/definition_store.go#L585) ever changes.

Reply:
Sounds good to me